packages/connectivity/src/scp-cf/cache.ts

```diff
@@ -86,7 +79,7 @@ export class Cache<T> implements CacheInterface<T> {
   set(key: string | undefined, item: CacheEntry<T>): void {
     if (key) {
       const expires =
-        item.expires ?? inferExpirationTime(this.defaultValidityTimeInMs);
+        item.expires ?? inferExpirationTime(this.defaultValidityTime);
```

Comment:
[req] Please adjust the signature of `inferExpirationTime` to not allow using `undefined`. Maybe consider adding it as a private method to the class, and rename it to something like `inferDefaultExpirationTime`. Then you don't need to pass `this.defaultValidityTime`. This is anyways the only place this is used, if I'm not mistaken.

Reply:
Thanks for the suggestion! I updated this part as suggested.